### ISSUE TITLE
Flickering Mihon Logo in About Page

### ISSUE THREAD

**Issue by Sanrax** [mihonapp/mihon#147]
### Steps to reproduce

Open About screen, watch the logo displayed at the top. After a couple seconds, it starts to flicker. I've tested this on a Motorola Edge 2020 and Moto G Stylus 2023, happens on both devices.

### Expected behavior

Logo should be displayed clearly without flicker.

### Actual behavior

Flickering can be seen on the logo when viewing the about screen.

https://github.com/mihonapp/mihon/assets/61475057/80b2eebd-4139-4a68-8f2e-2b80b29eb24a

### Mihon version

0.16.1

### Android version

Android 11, Android 13

### Device

Motorola Edge 2020, Moto G Stylus 2023

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.1](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by AntsyLich** [comment]
Can't reproduce on my phone or the android emulator

**Comment by Sanrax** [comment]
> Can't reproduce on my phone or the android emulator

I think I figured out why. It doesn't happen on my Moto G Stylus, so I probably just remembered wrong.

It did happen on my Moto Edge (that's the phone in the video), and it seemed to be caused by the phone's edge cutting feature where it doesn't allow app content to be displayed on the waterfall sides of the display. This feature is specific to this exact Motorola phone, as I don't believe any other Moto phones are like this one.

Anyway, turning that off fixed the issue, so I'll close this. (and I used to have it off in Tachiyomi as well, as I preferred to have the entire screen be used to display manga.)
